### ISSUE TITLE
What the learning rate should be?

### ISSUE THREAD

**Issue by ChaoLi977** [milesial/Pytorch-UNet#42]
Hi, I use the default learning rate which is 0.1 to train the U-net with my private dataset, but the resulted model is bad. I am wondering if the learning rate is too large. So, what the learning rate should be? Anyone know?

**Comment by milesial** [comment]
There is no magic number, it depends on the data. Try lower values like 0.3, 0.1, 0.05, ... plotting the loss curve would help you.
